pkg/services/store/entity/sqlstash/folder_support.go

```diff
@@ -33,51 +34,48 @@ type folderInfo struct {
 // This will replace all entries in `entity_folder`
 // This is pretty heavy weight, but it does give us a sorted folder list
 // NOTE: this could be done async with a mutex/lock?  reconciler pattern
-func updateFolderTree(ctx context.Context, tx *session.SessionTx, tenant int64) error {
-	_, err := tx.Exec(ctx, "DELETE FROM entity_folder WHERE tenant_id=?", tenant)
+func updateFolderTree(ctx context.Context, tx *session.SessionTx, tenantId int64) error {
+	_, err := tx.Exec(ctx, "DELETE FROM entity_folder WHERE tenant_id=?", tenantId)
 	if err != nil {
 		return err
 	}
 
+	query := "SELECT guid,uid,folder,name,slug" +
+		" FROM entity" +
+		" WHERE kind=? AND tenant_id=?" +
+		" ORDER BY slug asc"
+	args := []interface{}{entity.StandardKindFolder, tenantId}
+
 	all := []*folderInfo{}
-	rows, err := tx.Query(ctx, "SELECT uid,folder,name,slug FROM entity WHERE kind=? AND tenant_id=? ORDER BY slug asc;",
-		entity.StandardKindFolder, tenant)
+	rows, err := tx.Query(ctx, query, args...)
 	if err != nil {
 		return err
 	}
+	defer func() { _ = rows.Close() }()
```

Comment:
does `rows.Close()` return an error? if so, should it be logged? (or maybe a comment why it's safe to ignore the error)

Reply:
I believe I just added that to satisfy the linter, but we don't actually care (and there may be a linter exception now anyway)